client/src/components/pages/admin/Submitted_Id/index.js

```diff
@@ -1,5 +1,518 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Helmet } from 'react-helmet';
+import Checkbox from '@material-ui/core/Checkbox';
+import PropTypes from 'prop-types';
+import { useHistory, Link } from 'react-router-dom';
+import Limitation from '../../../common/limitation';
+import Card from '../../../common/card';
+import Typography from '../../../common/Typography';
+import Button from '../../../common/Button';
+import backGround from '../../../../assets/images/backgroundDash.svg';
+import DashBar from '../../../dashboard/Tabs';
+import './index.css';
 
-const SubmittedId = () => <h1>SubmittedId </h1>;
+const applicantData = async (id) => {
+  const {
+    data: { user },
+  } = await axios.get(`/api/v1/dashboard/applicant/${id}`);
+  return user;
+};
+const setAccept = async (id, acceptedValue) => {
+  await axios.patch(`/api/v1/dashboard/applicant/${id}`, {
+    accepted: !acceptedValue,
```

Comment:
@alidah0 I think it would be easier to understand if this function did not "flip"`acceptedValue` and instead, you "flip" the accepted value down on line 489 before passing it in as an argument, like you are doing with `setAcceptedVal()`.
I think that would be less surprising, if you ever need to use this function in a different way than your current button-toggling behavior.